css-two-column-layout/styles.css

```diff
@@ -0,0 +1,102 @@
+.container {
+  align-items: center;
+  font-family: Karla, sans-serif;
+  padding: 10px;
+  justify-content: center;
+}
+
+.title {
+  text-align: center;
+  margin: auto;
+  width: 700px;
+}
+
+.title-text {
+  font-size: 55px;
+  letter-spacing: 1.5px;
+}
+
+.summary {
+  bottom: 15px;
+  line-height: 1.5rem;
+  position: relative;
+}
+
+.row {
+  display: flex;
+  justify-content: center;
+  margin: auto;
+  position: relative;
+  width: 1000px;
+}
+
+.ben-images {
+  position: relative;
+  width: 100px;
+  margin-right: 5px;
+}
+
+.img-right {
+  left: 50px;
+}
+
+.col {
+  position: relative;
+  flex: 50%;
+  margin-right: 20px;
+  padding-right: 60px;
+}
+
+.col-left {
+  float: left;
+  left: 55px;
+}
+
+.col-right {
+  left: 90px;
+}
```

Comment:
You should not be using `float` or offset values for layouts. You should be utilizing flexbox for page layouts